apps/studio/components/ui/AIAssistantPanel/AIAssistant.tsx

```diff
@@ -0,0 +1,642 @@
+          ),
+        },
+      })
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [messages.length])
+
+  return (
+    <div className={cn('flex flex-col', className)}>
+      <SheetHeader className="flex items-center justify-between py-3">
+        <div className="flex items-center gap-x-2">
+          <AiIconAnimation
+            allowHoverEffect
+            className="[&>div>div]:border-black dark:[&>div>div]:border-white"
+          />
+          <p>Assistant</p>
+        </div>
+        <AnimatePresence>
+          {hasMessages && (
+            <motion.div
+              initial={{ opacity: 0 }}
+              animate={{ opacity: 100 }}
+              exit={{ opacity: 0 }}
+              transition={{ duration: 0.2 }}
+            >
+              <Button type="default" disabled={isLoading} onClick={() => onResetConversation()}>
+                Reset conversation
+              </Button>
+            </motion.div>
+          )}
+        </AnimatePresence>
+      </SheetHeader>
+
+      <SheetSection
+        className={cn(
+          'flex-grow flex flex-col items-center !p-0',
+          hasMessages ? 'justify-between h-[90%]' : 'justify-center h-full'
+        )}
+      >
+        {hasMessages && (
+          <motion.div
+            initial={{ height: 0 }}
+            // animate={{ height: '100%' }}
+            transition={{ duration: ANIMATION_DURATION }}
+            className="w-full overflow-auto flex-1"
+          >
+            {messages.map((m, index) => {
+              const isFirstUserMessage =
+                m.role === 'user' && messages.slice(0, index).every((msg) => msg.role !== 'user')
+
+              return (
+                <Message
+                  key={`message-${m.id}`}
+                  name={m.name}
+                  role={m.role}
+                  content={m.content}
+                  createdAt={new Date(m.createdAt || new Date()).getTime()}
+                  isDebug={(m as MessageWithDebug).isDebug}
+                  context={contextHistory[m.id]}
+                  onDiff={(diffType, sql) => onDiff({ id: m.id, diffType, sql })}
+                  // isSelected={selectedMessage === m.id}
+                >
+                  {isFirstUserMessage && !includeSchemaMetadata && true && (
+                    <Alert_Shadcn_>
+                      <AlertDescription_Shadcn_ className="flex flex-col gap-4">
+                        <span>
+                          Quick reminder that you're not sending project metadata with your queries.
+                          By opting into sending anonymous data, Supabase AI can improve the answers
+                          it shows you.
+                        </span>
+                        <Button
+                          type="default"
+                          className="w-fit"
+                          // onClick={() => setIsConfirmOptInModalOpen(true)}
```

Comment:
Still needed?
```suggestion
```

Reply:
oh good catch this! i need to set up if the user didnt opt into sending metadata